packages/devcmd-cli/src/index.ts

```diff
@@ -48,15 +48,42 @@ async function isDir(path: string): Promise<boolean> {
   }
 }
 
+async function startProcess(command: string, args: Array<string>, dirPath: string): Promise<number> {
+  const spawnOptions: SpawnOptions = {
+    stdio: 'inherit',
+    cwd: dirPath
+  };
+
+  const processPromise: Promise<number> = new Promise<number>((resolve, reject) => {
+    const processInstance: ChildProcess = spawn(command, args, spawnOptions);
+
+    processInstance.on("error", (err: Error): void => {
+      // The 'error' event gets emitted when the process couldn't be spawned, killed or communicated with
+      reject(err);
+    });
+
+    processInstance.on("close", (code: number): void => {
+      if (code === 0) resolve(code);
+      else reject(code);
+    });
+  });
+
+  return processPromise;
+}
+
+function isError(err: any): err is Error {
+  return err instanceof Error;
```

Comment:
minor: this doesn't have to be a function, you can just use `err instanceof Error` inline on line 87 and TS will correctly let you use the properties :)

Reply:
Good point, i thought this might be easier to read, but it doesnt really make a difference!